<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_component</class>
 <widget class="QWidget" name="fm_component">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QToolButton" name="tb_component">
         <property name="text">
          <string>...</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="la_component">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Component</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="fr_component">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="la_name">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="la_instance">
            <property name="text">
             <string>Instance:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="la_container">
            <property name="text">
             <string>Container:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cb_container"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_instance"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
